### PR TITLE
synapse, regen azure-synapse-artifacts with new spec commit

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
fix https://github.com/Azure/azure-sdk-for-java/issues/27497

code change due to updated spec is commit: https://github.com/Azure/azure-sdk-for-java/pull/27498/commits/d0a2f68d77dc257f930081a03a7a3ddadd3cd816

prepare release for azure-synapse-artifacts 1.0.0-beta.9 as HLC.

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
Same description in readme seems not correct from day1

> ### General
> Azure Synapse Artifacts clients raise exceptions. For example, if you try to retrieve an artifact after it is deleted a `404` error is returned, indicating the resource was not found. In the following snippet, the error is handled gracefully by catching the exception and displaying additional information about the error.
> 
> ```java
> try {
>     PipelineResource deletedPipeline = client.getPipeline(pipelineName);
> } catch (ResourceNotFoundException e) {
>     System.out.println(e.getMessage());
> }
> ```

There is no `ResourceNotFoundException` in code.

**Review by wonner — APPROVED** [review]

File: sdk/synapse/azure-analytics-synapse-artifacts/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,25 @@
 # Release History
 
-## 1.0.0-beta.9 (Unreleased)
+## 1.0.0-beta.9 (2022-03-08)
 
 ### Features Added
 
+- Added `ScriptActivity`.
+- Added `TeamDeskLinkedService`, `QuickbaseLinkedService`, `SmartsheetLinkedService`, `ZendeskLinkedService`.
```
Comment (srnagar) [inline sdk/synapse/azure-analytics-synapse-artifacts/CHANGELOG.md:8]: nit: adding some more details may help here. Not sure why we have `ScriptActivity` in a separate line and the rest of the `*Service` types in one line.

**Review by srnagar — APPROVED** [review]

File: sdk/synapse/azure-analytics-synapse-artifacts/CHANGELOG.md (hunk 1 above)
Comment (weidongxu-microsoft) [reply sdk/synapse/azure-analytics-synapse-artifacts/CHANGELOG.md:8]: Sure. Class on 2 lines is added to different polymorphic base class. I will update the changelog.

**Pull request merged**